api/src/test/java/io/opentelemetry/trace/propagation/HttpTraceContextTest.java

```diff
@@ -189,7 +191,7 @@ public void extract_NotSampledContext_EmptyTraceState() {
   public void extract_NotSampledContext_TraceStateWithSpaces() {
     Map<String, String> carrier = new LinkedHashMap<>();
     carrier.put(TRACEPARENT, TRACEPARENT_HEADER_NOT_SAMPLED);
-    carrier.put(TRACESTATE, "foo=bar   ,    bar=baz");
+    carrier.put(TRACESTATE, TRACESTATE_NOT_DEFAULT_ENCODING_WITH_SPACES);
```

Comment:
note: this test was actually not testing the right thing, since the equals was missing the trace state.